src/SeqViz/CommonTypes.ts

```diff
@@ -0,0 +1,30 @@
+import * as React from "react";
+
+export type InputRefFuncType = <T>(id: string, ref: unknown) => React.LegacyRef<T>;
+
```

Comment:
uppercase file names indicate the file exports a React component. This could be `shared.ts` or `common.ts` or something like that